x/staking/client/rest/utils.go

```diff
@@ -111,6 +111,11 @@ func queryValidator(cliCtx context.CLIContext, endpoint string) http.HandlerFunc
 		vars := mux.Vars(r)
 		bech32validatorAddr := vars["validatorAddr"]
 
+		_, page, limit, err := rest.ParseHTTPArgsWithLimit(r, 0)
```

Comment:
Default limit of 0? We should provide something sensible.

Reply:
I used the approach is used in the other parts of the code, there is only one case that the limit has a value different than 0, is in client/rpc/validators.go

I can set it to 100 here, but we could think to add it to issue #6191 to unify all cases.